src/xiana/route/helpers.clj

```diff
@@ -16,3 +23,73 @@
           (assoc :error e)
           (assoc :response
                  {:status 500 :body "Internal Server error"})))))
+
+(defonce all-methods
```

Comment:
It should be part of `xiana.swagger` ns, isn't it?

Reply:
I was seeing the future again, and experimenting with more advanced and growth code-base. So, for now, fixed.